kernel/src/hil/touch.rs

```diff
@@ -0,0 +1,83 @@
+//! Interface for touch input devices
+
+use crate::ReturnCode;
+
+#[derive(Debug, Copy, Clone)]
+pub enum TouchStatus {
```

Comment:
We generally require doc comments (`///`) for every `pub` entity and interface in HILs, even for the seemingly obvious stuff, so that the docs (https://docs.tockos.org/kernel/hil/index.html) render nicely.